### DISCUSSION TITLE
starting Carla in Standalone after building from src without CarlaUE4.sh

### DISCUSSION THREAD

**Discussion by isochor** [carla-simulator/carla#6791] [General]
Hello, I build Carla from source code and when I execute make launch or make launch-only it works. I want to start carla in standalone mode (without unreal editor) and I found that I have to use this ocmmand: 

`./CarlaUE4.sh -quality-level=Epic -windowed -ResX=800 -ResY=600`

however, since I build Carla from src I don't have CarlaUE4.sh. I took a look at the shell script I'd had to run but it doesn't make sense since only thing this shell does is to start the binary CarlaUE4-Linux-Shipping.target which I don't have neither. But in the folder where this CarlaUE4-Linux-Shipping should be, there is no binary nearly as big as this file, so I can't change the shell manually.

Thanks

**Comment by kkirchheim** [comment]
In case anyone is still looking for a solution: you can use `make package` to create the shipping version of CARLA that works without the editor. This also includes the `CarlaUE4.sh`.
